reflex/components/radix/primitives/accordion.py

```diff
@@ -194,9 +194,9 @@ class AccordionItem(AccordionComponent):
     disabled: Var[bool]
 
     # The header of the accordion item.
-    header: Optional[Union[Component, str, Var[Union[Component, str]]]] = None
+    header: Var[Union[Component, str]] = Var.create(None)
```

Comment:
i think the default is `None`
```suggestion
    header: Var[Union[Component, str]]
```